google-cloud-bigquery-data_transfer/samples/Gemfile

```diff
@@ -0,0 +1,29 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+source "https://rubygems.org"
+
+if ENV["GOOGLE_CLOUD_SAMPLES_TEST"] == "master"
+  gem "google-cloud-bigquery-data_transfer", path: "../../google-cloud-bigquery-data_transfer"
```

Comment:
Should I add the actual client here as well?

```
gem "google-cloud-bigquery-data_transfer-v1", path: "../../google-cloud-bigquery-data_transfer-v1"
```

Reply:
Yes

Reply:
Done